lib/fxa-geodb.js

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var DEFAULTS = require('./defaults');
+var ERRORS = require('./errors');
+var maxmind = require('maxmind');
+var Promise = require('bluebird');
+
+module.exports = function (options) {
+  'use strict';
+
+  options = options || {};
```

Comment:
@TDA these options are not documented anywhere, I suggest JSDOC them and generate an MD file, so we don't have to manually update docs all the time

Example: https://github.com/vladikoff/foxfire/blob/master/docs.md and https://github.com/vladikoff/foxfire/blob/master/Gruntfile.js#L4